docs/Contribute/Agenda.md

```diff
@@ -3,23 +3,13 @@
 BTCPay Server operates as an open-organization. All of our meetings where we discuss the development, design, documentation or other topics are opened for anyone to join in and participate. To get involved, you can subscribe to our open-calendar and stay in the loop on what we're working on, learn, ask questions or just join us in building open-source bitcoin payment rails.
 Follow the steps below to add the BTCPay Server `Open Community Calls` calendar to your own. 
 
-## Upcoming calls
-
-| Topic        | Direct link                                                                                        |
-|--------------|-----------:|
-| Development  | [Link](https://github.com/btcpayserver/organization/issues?q=is%3Aissue+is%3Aopen+%27Dev%27)       |
-| Design       | [Link](https://github.com/btcpayserver/organization/issues?q=is%3Aissue+is%3Aopen+%27Design%27+)   |
-| Documentation| [Link](https://github.com/btcpayserver/organization/issues?q=is%3Aissue+is%3Aopen+%27Marketing%27) |
-
 ## BTCPay Server Open Community Calls calendar
 
-Our calendar (an .iCal file) includes upcoming calls. Qualified issues are automatically added to the calendar, which you can then subscribe to via your favorite calendar tool.
+Qualified issues are automatically added to the calendar, which you can then subscribe to via your favorite calendar tool.
 
-* Copy this URL: https://raw.githubusercontent.com/btcpayserver/organization/calendar/events.ical
-* In Apple Calendar (desktop), use File -> New calendar subscriptions
-* In Google Calendar (web), click the small + icon next to Other calendars in the sidebar and select From URL
-* Paste the URL and save. New events (if there are any scheduled) should show up right away
-* In Thunderbird (desktop), New Calendar > On my network > iCalendar (ICS) and paste the calendar link in the location field.
+* In Apple Calendar (desktop), use File -> New calendar subscriptions and paste this [iCal format public address](https://calendar.google.com/calendar/ical/1m0qsj2dg72pisr9ku91qi1v28%40group.calendar.google.com/public/basic.ics)
```

Comment:
I am not 100% sure the process is the same with google link, have we tried to replicate it and assured it's works?

Reply:
Before submitting, I have tested all the links with the built-in Apple "Calendar" app, Thunderbird's Calendar for Mac and the google calendar web.

Reply:
Fantastic, thanks for doing that, just wanted to be 100% sure :)